### ISSUE TITLE
No "Socket.onAny(Event)" function in Java

### ISSUE THREAD

**Issue by aqibbangash** [socketio/socket.io-client-java#475]
Unlike iOS library, I'm unable to find onAny(Event) function in Java Library. 

This demands reframing the whole architecture for an app that is supposed to be developed for Android and iOS both.

**Comment by Jedidiah** [comment]
@aqibbangash Realise this was a few years ago and might not be relevant anymore but I was just trying to do the same thing and I think what might help you is `.on(io.socket.client.Socket.EVENT_MESSAGE, new Emitter.Listener() { ...`

**Comment by michalmonday** [comment]
Is there any other way to achieve that? I get the following error with solution above:
> EVENT_MESSAGE is not public in io.socket.client.Socket; cannot be accessed from outside package

**Comment by darrachequesne** [comment]
Here we go!

```java
socket.onAnyIncoming(new Emitter.Listener() {
    @OverRide
    public void call(Object... args) {
        // ...
    }
});
```

Included in version `2.1.0`.

**Comment by irshadir7** [comment]
@darrachequesne after updated to 2.1.0 socket is not connecting . getting server error. may i know the reason? kindly help me pls
